src/components/Home/components/Weather/components/CurrentWeather/CurrentWeather.tsx

```diff
@@ -1,16 +1,18 @@
 import * as React from 'react';
+
+import * as moment from 'moment';
 import { CurrentWeatherType } from '../../Weather'
 
 
 export class CurrentWeather extends React.Component<CurrentWeatherType, {}> {
```

Comment:
this function has no state, better to write it as a functional component instead

Reply:
Gonna keep it as is. Might be statefull depending on what features we are interested in implementing